app/commands/install.go

```diff
@@ -100,14 +100,14 @@ func extractCMD(dir, archiveName string) error {
 	if runtime.GOOS == "windows" {
 
 		ext = ".exe"
-		err := archiver.Zip.Open(dir+"/"+archiveName, dir)
+		err := archiver.DefaultZip.Unarchive(dir+"/"+archiveName, dir)
```

Comment:
is this better than zip.Open ?

Reply:
That method no longer exists. When I switched to go mod, it must've pulled in a newer version

Reply:
...and wouldn't compile